.circleci/config.yml

```diff
@@ -14,6 +22,16 @@ jobs:
       - checkout
       - run:
           command: npm ci
+      - <<: *run_env_setup
+      - run:
+          name: Install git secrets and configure minnie-kenny.sh
+          command: |
+            git clone https://github.com/awslabs/git-secrets.git
+            pushd git-secrets >/dev/null
+            git checkout ad82d68ee924906a0401dfd48de5057731a9bc84
+            popd >/dev/null
+            ./minnie-kenny.sh -f
+      - <<: *run_env_setup
```

Comment:
What does this command do?

Reply:
`run_env_setup` is defined on line 7 and is used to make sure that `git secrets` is find-able (by adding it to the PATH environmental variable